backend/functions/mountain_service.py

```
@@ -0,0 +1,119 @@
+        """
+        self.logger = logger
+
+    def _collect_mountain_data(self) -> bytes:
+        """
+        Downloads the ZIP file containing mountain data and returns its content as bytes.
+        """
+        # Declare the URL for the ZIP file download
+        url = "https://www.hill-bagging.co.uk/dobih-downloads/hillcsv.zip"
+
+        # Download the zip file stream from the URL
+        response = requests.get(url, stream=True)
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)